### PR TITLE
Minor doc review

### PR DESCRIPTION
**Pull request by phil-davis**
various little bits of wording enhancement while I am reading.

### CODE REVIEW

File: guides/managing-sessions.rst [hunk 1]
```diff
@@ -43,7 +43,7 @@ one single container object:
 
 .. note::
 
-    Mink will even lazy-start your sessions when needed (on first ``getSession()``
+    Mink will even lazy-start your sessions when needed (on the first ``getSession()``
```
Comment (stof) [inline guides/managing-sessions.rst]: actually, this doc is outdated. The Mink class does not ever start session anymore. It is a simple registry. Sessions themselves are now starting automatically when you visit a page for the first time.

Comment (phil-davis) [inline guides/managing-sessions.rst]: "real" doc content updates can easily be done in separate PRs. I have not attempted to check the truth of the statements, just fixed up some grammar.
